static-site/FAQ.md

```diff
@@ -192,30 +190,38 @@ There is a detailed Confluence page that contains a more in-depth overview of th
 
 60 days
 
-#### When a claim is updated, will a new claim with a different claim ID be created and the previous claim marked as cancelled? OR will the existing claim be updated and the status remain unchanged?
+#### When a claim is updated, will a new claim with a different claim ID be created and the previous claim marked as cancelled? Or will the existing claim be updated and the status remain unchanged?
 
 It's more likely the claim will get updated and the status remains the same. The status will only change to cancelled if the final action field we receive from CCW changes.
 
 ### FHIR Mapping
 
 #### Does the BFD FHIR API send Part A claims data for care received in Critical Access Hospitals and teaching hospitals for Part C enrollees?
 
-No. This information is not provided as part of the CCW-based BFD FHIR API.
+No. This information is not provided as part of the CCW-based BFD FHIR API. While we do have Part C enrollee data, our APIs focus is on Fee for Service Medicare (A, B, & D) and not Medicare Advantage (Part C).
 
 #### Does the BFD FHIR API provide information on benefit details such as deductibles, copays, coinsurance, etc.?
 
-In general, we have somewhat limited information available for deductibles. For Part A claims, we have deductible-related data elements for Inpatient and Skilled Nursing Facility (SNF) claims: 
+In general, BFD has somewhat limited information available for deductibles. For Part A claims, BFD has deductible related data elements for Inpatient and Skilled Nursing Facility (SNF) claims:
```

Comment:
I think the hyphen here is correct since "deductible-related" is a compound modifier
```suggestion
In general, BFD has somewhat limited information available for deductibles. For Part A claims, BFD has deductible-related data elements for Inpatient and Skilled Nursing Facility (SNF) claims:
```